src/main/java/sg/ncl/MainController.java

```diff
@@ -1974,9 +1983,10 @@ public String checkApplyTeamInfo(
         mainObject.put("team", teamFields);
         teamFields.put("name", teamPageApplyTeamForm.getTeamName());
         teamFields.put("description", teamPageApplyTeamForm.getTeamDescription());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'DESCRIPTION' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)